cms/templates/settings.html

```diff
@@ -150,9 +152,10 @@ <h2 class="title-2">${_("Credit Eligibility Requirements")}</h2>
 
                   % if 'reverification' in credit_requirements:
                     <li class="field text is-not-editable" id="credit-reverification-requirements">
-                      <label for="reverification-requirements">${_("Successful In Course Reverification")}</label>
+                      <label>${_("Successful In Course Reverification")}</label>
                         % for requirement in credit_requirements['reverification']:
                           ## Translators: 'Access to Assessment 1' means the access for a requirement with name 'Assessment 1'
+                          <label for="${requirement['name']}" class="sr">${_('In Course Reverification {counter}').format(counter=loop.index+1)}</label>
```

Comment:
This would be a little easier for translators to understand if you used `number` instead of `counter`.
```python
${_('In-course Reverification {number}').format(number=loop.index+1)}
```